examples/sources/open_telemetry_logs.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+  required_providers {
+    mezmo = {
+      source = "registry.terraform.io/mezmo/mezmo"
+    }
+  }
+  required_version = ">= 1.1.0"
+}
+
+provider "mezmo" {
+  auth_key = "my secret"
+}
+
+resource "mezmo_pipeline" "pipeline1" {
+  title = "My pipeline"
+}
+
+resource "mezmo_open_telemetry_logs_source" "source1" {
+  pipeline_id = mezmo_pipeline.pipeline1.id
+  title       = "My Open Telemetry Logs source"
+  description = "This receives data from Open Telemetry"
+}
+
+resource "mezmo_open_telemetry_logs_source" "shared_source" {
+  pipeline_id      = mezmo_pipeline.pipeline1.id
+  title            = "A shared Open Telemetry Logs source"
+  description      = "This source uses the same data as source1"
+  gateway_route_id = mezmo_open_telemetry_logs_source.source1.gateway_route_id
+}
```

Comment:
This is a note to all of us--no need to change it here.  We should stop doing shared routes on all sources.  We should have a section dedicated to doing shared sources and have examples for that.  I think this example here takes away from the simplicity of some of these sources.